app/helpers/orders_helper.rb

```diff
@@ -4,4 +4,16 @@ module OrdersHelper
   def total_price(order)
     order.order_histories.reduce(0) { |sum, value| sum + value.current_price * value.quantity }
   end
+
+  def calculate_status(string)
+    return true if string == Order::ACCEPT
+
+    false
```

Comment:
```suggestion
    return string == Order::ACCEPT
```
dc ko cu =))
nếu trả về true false thì em thêm dấu `?` vào method nhé